.github/workflows/main.yaml

```diff
@@ -1,5 +1,8 @@
 name: Build
-on: [push]
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
We specify types to omit 'edit', which will rerun ci whenever the description of PR changes